### PR TITLE
Chapter 8

### PR DESCRIPTION
**Pull request by thaidoandat**
image
link questions: https://docs.google.com/document/d/1GFPo8Xc29CrcNkbCtUSq69EirjnvgB_knkBdE4a0pgE/edit

### CODE REVIEW

**Comment by thaidoandat** [comment]
ready

**Comment by ghost** [comment]
fix conflict em ơi

**Comment by ngocvt-0484** [comment]
commented

**Comment by thaidoandat** [comment]
ready

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for :session, url: login_path do |f| %>
+      <%= f.label :email %>
+      <%= f.email_field :email, class: "form-control" %>
+      <%= f.label :password %>
+      <%= f.password_field :password, class: "form-control" %>
+      <%= f.submit t("layouts.header.login"), class: "btn btn-primary" %>
+    <% end %>
+    <p>New User? <%= link_to t("static_pages.home.signup"), signup_path %></p>
```
Comment (ghost) [inline app/views/sessions/new.html.erb]: i18n

**Review by ghost — COMMENTED** [review]
cmt nhỏ fix rồi merged em nhé

**Pull request merged**